requirements.in

```diff
@@ -21,7 +21,7 @@ Pillow==9.4.0
 python-dateutil
 django-storages
 wheel
-cryptography
+cryptography>=41.0.0
```

Comment:
is there a specific need to pin this? wouldn't a regular pip-compile --upgrade suffice?

Reply:
Mostly just my own pip-compile ignorance. I should prep myself a cheat sheet -- I'm just so used to pinning things. I can re-do it once I'm at my machine.